src/utils/get-package-base.ts

```diff
@@ -10,9 +11,10 @@ module.exports = function (id) {
     if (pkgNameMatch)
       return id.substr(0, pkgIndex + 13 + pkgNameMatch[0].length);
   }
-};
+  return undefined;
```

Comment:
There was an implicit return here so do we expect this to be undefined in some cases?

Reply:
Yes this is correct.